go.mod

```diff
@@ -20,17 +21,10 @@ require (
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/component-base v0.21.1
 	k8s.io/klog/v2 v2.8.0
-	kubevirt.io/client-go v0.36.0
+	kubevirt.io/api v0.0.0-20211117075245-c94ce62baf5a
 	sigs.k8s.io/cluster-api v0.3.11-0.20210525210043-6c7878e7b4a9
 	sigs.k8s.io/controller-runtime v0.9.0-beta.5
 	sigs.k8s.io/kind v0.11.0
 )
 
-replace (
-	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20191219222812-2987a591a72c
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
-	k8s.io/client-go => k8s.io/client-go v0.21.1
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.2
-	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-e85c7b244fd2
-)
+replace k8s.io/client-go => k8s.io/client-go v0.21.1
```

Comment:
@nirarg do you want to also do the cdi dependency replace which brings in https://github.com/kubevirt/containerized-data-importer/pull/2032 which further reduces the dependencies?